### DISCUSSION TITLE
[Feature Request] Telegram lock down accepted chatids to allow only specific people to interact with the bot

### DISCUSSION THREAD

**Discussion by Tigger2014** [arendst/Tasmota#10765] [Ideas]
**Have you looked for this feature in other issues and in the docs?**
Yes

**A clear and concise description of what the problem is.**
Telegram bots do not support authentication so anyone can talk to any bot. This would mean that anyone could talk to a bot used for Tasmota and control the device.

**Describe the solution you'd like**
A new option such as tmAllowedIds which can be passed multiple chatids that are allowed to perform actions on Tasmota. Messages from other chatIDs should be discarded.

Thanks

**Comment by mosix76** [comment]
It should very usefull to increase the telegram security on Tasmota
